CMakeLists.txt

```diff
@@ -105,3 +107,34 @@ if(IMNODES_EXAMPLES)
         target_link_libraries(hello X11 Xext GL)
     endif()
 endif()
+
+install(
+  TARGETS imnodes
+  EXPORT imnodes_target
+  ARCHIVE COMPONENT Development DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  LIBRARY COMPONENT Runtime
+          NAMELINK_COMPONENT Development
+          DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  RUNTIME COMPONENT Runtime DESTINATION ${CMAKE_INSTALL_BINDIR})
+
+install(
+  FILES imnodes.h imnodes_internal.h
+  COMPONENT Development
+  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
```

Comment:
![image](https://github.com/Nelarius/imnodes/assets/9720532/cd6189ac-ce78-4422-b469-1411c288cbd5)


I'm surprised to see this, given that the documentation claims `"include"` is the default/fallback value for the variable: https://cmake.org/cmake/help/latest/command/install.html#signatures